src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1551,6 +1552,46 @@ internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers(string
             return _lazyEarlyAttributeDecodingMembersDictionary;
         }
 
+        private static Dictionary<ReadOnlyMemory<char>, ImmutableArray<Symbol>> ToNameKeyedDictionary(ImmutableArray<Symbol> symbols)
```

Comment:
dedicated copy the copmiler gets of this original extension method.  It's only used in this type, and always with the specialized ROM behavior.  So makes sense to just live here and not in a general place.